src/stylesheets/components/_article.scss

```diff
@@ -44,4 +44,41 @@ article {
       }
     }
   }
+
+  & + aside .block .list--pipe:last-child > li:last-child {
+    border-bottom: 0;
+  }
+
+  @media (min-width: $tablet) {
+
+    // These styles for `h2, p, ol, ul` tags are to align
+    // the baseline of each of those element's text with
+    // the baseline of the `h2` tag inside the `article`
+    // tag's ajacent sibling the `aside` tag.
+    //
+    // This also fixes the extra spacing of 18px a
+    // first-child 'h2' tag makes when inside a new
+    // `section` tag. In this scenario, this fix is only
+    // happening when the `h2` tag is inside the `article`
+    // tag. However, there is now extra space above the
+    // first-child `p, ol, ul` tags to align their
+    // first-line text baselines with the `h2` tag inside
+    // the `aside` tag.
+
+    h2:first-child {
+      margin-top: $base-spacing-unit-medium;
+    }
+    p:first-child,
```

Comment:
As mentioned above this could be a padding on the article itself if it is consistent.

Reply:
I have removed these styles completely and have added a convenient padding instead. Also, this may look odd on pages that have a paragraph or list tag as the first child of the article tag. However, assuming it's a requirement to add a heading 2 as the first child for all articles, which we will need to address to article editors and they will have to implement, then this new update is correct.